src/components/TitleItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <h1>
+    {{ props.title }}
+  </h1>
+</template>
+
+<script setup lang="ts">
+  const props = defineProps(["title"])
+</script>
+
+<style scoped lang="postcss">
+  h1 {
```

Comment:
Alright, a couple of things to mention here:
1. In general, try to avoid using tags as selectors. that's a bad practice because of multiple reasons. If you want to know more why, feel free to ping me.
2. We are using Tailwindcss in the project and literally everything you've done here can be done in Tailwind. So please use that. If you have any questions regarding Tailwind, you can check out the documentation in the README of the project or just ping me.
3. Hardcoded color should be also moved in the Tailwind and used as a class, or, at the very least, used as a custom class ```color-[#34495e]```